src/lib_rules.ml

```diff
@@ -377,12 +413,16 @@ module Gen (P : Install_rules.Params) = struct
          else
            acc)
      in
+     let deprecated_modules = Module.Name.Map.values wrapped_compat in
```

Comment:
Just to be consistent, let's use the `wrapped_compat` terminology here as well. This will make it easier to understand this code in the future

Reply:
That was just an omission. I fixed this.